### DISCUSSION TITLE
FAQ Guidance Updates

### DISCUSSION THREAD

**Discussion by shaselton-usds** [CMSgov/price-transparency-guide#713] [Announcements]
Please see this newly uploaded [FAQ](https://www.cms.gov/files/document/faqs-about-affordable-care-act-implementation-part-61.pdf) from the Departments of Labor, Health and Human Services, and the Treasury regarding the prescription drug machine-readable file requirement.

**Comment by santaviccap** [comment]
@shaselton-usds  - when do we expect development to restart on the RX file?

**Reply by santaviccap** [reply]
Hi @shaselton-usds - checking back in here, is there even an approximate time-frame that can be shared around the re-start of the RX Schema?  Do we expect that to begin at some point in the first half of 2024?  Second half of 2024?  

Thanks so much for any additional info that can be shared.

**Reply by santaviccap** [reply]
@shaselton-usds  - squeaky wheel gets the oil!  Any news for those of us super excited about prescription drug MRF's?

**Reply by nmayle** [reply]
It has been more than four months since September 27, 2023 when the FAQ was posted stating that:

> The Departments intend to develop technical requirements and an implementation timeline in future guidance

It would be very helpful to have a timeline so the payers and the users of this data can begin to schedule resources to create and/or handle this file.

**Reply by JR-CMS** [reply]
As stated in FAQ 61 (https://www.cms.gov/files/document/faqs-about-affordable-care-act-implementation-part-61.pdf), the Departments will release technical requirements and an implementation timeline in future guidance that will provide plans and issuers with sufficient notice to develop their prescription drug files.

**Comment by sandyl1879** [comment]
Hi @shaselton-usds , I am wondering if an update will be made to this [page] (https://github.com/CMSgov/price-transparency-guide/tree/master/schemas/prescription-drugs) now that the new FAQ is out.

**Reply by ACKnox** [reply]
@sandyl1879 are you referencing the FAQ that was published on September 27, 2023, FAQ Part 61?

**Reply by sandyl1879** [reply]
@ACKnox yes, since the FAQ came out many believe they now have to publish the prescription drug file and there is no schema.

**Reply by ACKnox** [reply]
Thank you, just wanted to clarify we were on the same page with the FAQ.
